### ISSUE TITLE
Error if no --etag option provided

### ISSUE THREAD

**Issue by farwayer** [import-io/s3-deploy#4]
```
Upload error: NetworkingError: `value` required in setHeader("x-amz-meta-ETag", value). (NetworkingError: `value` required in setHeader("x-amz-meta-ETag", value)
```

**Comment by hartz89** [comment]
From the README.md

> To automatically fill this header with MD5 hash of the file, just use --tag parameter without any value. Internally the tool will generate MD5 hash of the content and will set it as the ETag header value. By default this parameter is undefined.

However, I tried just using `--tag` and got the same error.

**Comment by hartz89** [comment]
Hah!  There was a typo in the docs; I've already put in a PR to fix it but if you provide `--etag` with no value, it will generate the ETag header value for you.

**Comment by domderen** [comment]
This should now be fixed in `0.5.1` version of the package.

Sorry for the late response.
